### PR TITLE
Added new SDK versions to changelog

### PR DESCRIPTION
**Pull request by phgoncalves**
## Description
Added the new SDKs' versions to the changelog.

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [x] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [x] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/changelog/current-year.es.md [hunk 1]
```diff
@@ -4,6 +4,80 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+
+> CHANGELOG
+>
+> Nueva versión SDK PHP
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK PHP
+
+Lanzamos una nueva versión del SDK Java 2.1.
```
Comment (juones) [inline guides/changelog/current-year.es.md:54]: Lanzamos una nueva versión del SDK PHP 2.1.0.

File: guides/changelog/current-year.pt.md [hunk 2]
```diff
@@ -4,6 +4,79 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+
+> CHANGELOG
+>
+> Nova versão SDK PHP
+>
+> NEW_VERSION: NOVA VERSÃO
+>
+> PRODUCT: SDK PHP
+
+Lançamos uma nova versão do SDK Java 2.1.
```
Comment (juones) [inline guides/changelog/current-year.pt.md:54]: Lançamos uma nova versão do SDK PHP 2.1.0.

File: guides/changelog/current-year.en.md [hunk 3]
```diff
@@ -4,6 +4,80 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+
+> CHANGELOG
+>
+> New version of SDK NodeJS
+>
+> NEW_VERSION: NEW VERSION
+>
+> PRODUCT: SDK NodeJS
+
+We have released a new version of SDK Java 1.3.2.
```
Comment (juones) [inline guides/changelog/current-year.en.md:19]: We have released a new version of SDK NodeJS 1.3.2.

File: guides/changelog/current-year.es.md [hunk 4]
```diff
@@ -4,6 +4,80 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+
+> CHANGELOG
+>
+> Nueva versión SDK NodeJS
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK NodeJS
+
+Lanzamos una nueva versíon del SDK Java 1.3.2.
```
Comment (juones) [inline guides/changelog/current-year.es.md:19]: Lanzamos una nueva versíon del SDK NodeJS 1.3.2.

File: guides/changelog/current-year.pt.md [hunk 5]
```diff
@@ -4,6 +4,79 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+
+> CHANGELOG
+>
+> Nova versão SDK NodeJS
+>
+> NEW_VERSION: NOVA VERSÃO
+>
+> PRODUCT: SDK NodeJS
+
+Lançamos uma nova versão do SDK Java 1.3.2.
```
Comment (juones) [inline guides/changelog/current-year.pt.md:19]: Lançamos uma nova versão do SDK NodeJS 1.3.2.

File: guides/changelog/current-year.pt.md [hunk 6]
```diff
@@ -4,6 +4,79 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+
+> CHANGELOG
+>
+> Nova versão SDK Java
+>
+> NEW_VERSION: NOVA VERSÃO
+>
+> PRODUCT: SDK JAVA
+
+Lançamos uma nova versão do SDK Java 1.7.
```
Comment (juones) [inline guides/changelog/current-year.pt.md:37]: Lançamos uma nova versão do SDK Java 1.7.0.

File: guides/changelog/current-year.pt.md [hunk 7]
```diff
@@ -4,6 +4,79 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+
+> CHANGELOG
+>
+> Nova versão SDK .NET
+>
+> NEW_VERSION: NOVA VERSÃO
+>
+> PRODUCT: SDK .NET
+
+Lançamos uma nova versão do SDK .NET 1.8.
```
Comment (juones) [inline guides/changelog/current-year.pt.md:71]: Lançamos uma nova versão do SDK .NET 1.8.0.

File: guides/changelog/current-year.en.md [hunk 8]
```diff
@@ -4,6 +4,80 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+
+> CHANGELOG
+>
+> Nueva versión SDK Java
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK JAVA
+
+We have released a new version of SDK Java 1.7.
```
Comment (juones) [inline guides/changelog/current-year.en.md:37]: We have released a new version of SDK Java 1.7.0.

File: guides/changelog/current-year.en.md [hunk 9]
```diff
@@ -4,6 +4,80 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+
+> CHANGELOG
+>
+> Nueva versión SDK .NET
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK .NET
+
+We have released a new version of SDK .NET 1.8.
```
Comment (juones) [inline guides/changelog/current-year.en.md:71]: We have released a new version of SDK .NET 1.8.0.

File: guides/changelog/current-year.es.md [hunk 10]
```diff
@@ -4,6 +4,80 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+
+> CHANGELOG
+>
+> Nueva versión SDK Java
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK JAVA
+
+Lanzamos una nueva versión del SDK Java 1.7.
```
Comment (juones) [inline guides/changelog/current-year.es.md:37]: Lanzamos una nueva versión del SDK Java 1.7.0.

File: guides/changelog/current-year.es.md [hunk 11]
```diff
@@ -4,6 +4,80 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+
+> CHANGELOG
+>
+> Nueva versión SDK .NET
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK .NET
+
+Lanzamos una nueava versión del SDK .NET 1.8.
```
Comment (juones) [inline guides/changelog/current-year.es.md:71]: Lanzamos una nueava versión del SDK .NET 1.8.0.

Comment (mminestrelli) [reply guides/changelog/current-year.es.md:71]: nueva

File: guides/changelog/current-year.en.md (hunk 3 above)
Comment (mminestrelli) [inline guides/changelog/current-year.en.md:19]:
```suggestion
We have released a new version of SDK NodeJS 1.3.2.
```

File: guides/changelog/current-year.en.md [hunk 12]
```diff
@@ -4,6 +4,80 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+
+### July 7th
+
+> CHANGELOG
+>
+> New version of SDK NodeJS
+>
+> NEW_VERSION: NEW VERSION
+>
+> PRODUCT: SDK NodeJS
```
Comment (pamalv7) [inline guides/changelog/current-year.en.md:17]:
```suggestion
> PRODUCT: SDK NODEJS
```

File: guides/changelog/current-year.es.md [hunk 13]
```diff
@@ -4,6 +4,80 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+
+### 7 de julio
+
+> CHANGELOG
+>
+> Nueva versión SDK NodeJS
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK NodeJS
```
Comment (pamalv7) [inline guides/changelog/current-year.es.md:17]:
```suggestion
> PRODUCT: SDK NODEJS
```

File: guides/changelog/current-year.pt.md [hunk 14]
```diff
@@ -4,6 +4,79 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+
+### 7 de julho
+
+> CHANGELOG
+>
+> Nova versão SDK NodeJS
+>
+> NEW_VERSION: NOVA VERSÃO
+>
+> PRODUCT: SDK NodeJS
```
Comment (pamalv7) [inline guides/changelog/current-year.pt.md:17]:
```suggestion
> PRODUCT: SDK NODEJS
```

**Pull request merged**